server/api.py

```diff
@@ -93,23 +94,25 @@ def read_map(claims, id):
 
         Returns a HTTP response.
         """
-        email, user, map = None, None, None
+        email, user, game_map = None, None, None
         try:
             email = claims["email"]
             user = User.objects(email=email).first()
         except Exception as e:
             return malformed_request()
 
         try:
-            map = Map.objects.get(id=id, user=user)
+            game_map = GameMap.objects(id=id).first()
```

Comment:
just want to verify, the reason i used user=user and id=id was because another user could read other users maps given only the id and if you check if its the right user then that's not possible. is that still a risk or did we correct that fully?

Reply:
I can add that back in.